fe/fe-core/src/main/java/com/starrocks/privilege/GlobalFunctionPEntryObject.java

```diff
@@ -0,0 +1,116 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.privilege;
+
+import com.google.common.base.Objects;
+import com.google.common.base.Preconditions;
+import com.google.gson.annotations.SerializedName;
+import com.starrocks.catalog.Function;
+import com.starrocks.server.GlobalStateMgr;
+
+import java.util.List;
+
+public class GlobalFunctionPEntryObject implements PEntryObject {
+    @SerializedName(value = "f")
```

Comment:
Is this value too short?

Reply:
Short value can save space. Plus this short value scope is just inside of class `GlobalFunctionPEntryObject`, when we deserialize from json, the content is like

```
{"name": "GlobalFunctionPEntryObject", "data": {"f": ...}}}
```